services/nycrc/nycrc.service.js

```diff
@@ -50,9 +50,7 @@ export default class Nycrc extends ConditionalGithubAuthV3Service {
         .default('.nycrc'),
       // Allow the default threshold detection logic to be overridden, .e.g.,
       // favoring lines over branches:
-      preferredThreshold: Joi.string()
-        .optional()
-        .allow(...validThresholds),
+      preferredThreshold: Joi.string(),
```

Comment:
In this case, I chose to stay permissive, as we're manually enforcing a strict check lower down:
https://github.com/badges/shields/blob/d3a8267c6c54361b51a4626ab903336a5e13a576/services/nycrc/nycrc.service.js#L93